### DISCUSSION TITLE
Hide unselectable minutes

### DISCUSSION THREAD

**Discussion by RvmMim** [mskocik/svelty-picker#158] [Q&A]
Hi! I set minuteIncrement to 30 so the user can select only half hours but all unselectable minutes are still visible.
They can't be selected (the closest half hour is automatically selected) but they're visible.
Is there a way to hide them ?

**Comment by mskocik** [comment]
It's not possible ATM. Not even through CSS, because there is not specific "minute-view" class.

I could add css class for minute view. And then you could style it yourself. Would that solution be enough for you?

**Reply by RvmMim** [reply]
Thank you for your quick reply.
Would this minute-view class be applied only to minutes elements whose data-value is a multiple of minuteIncrement ?
If so then yes that would help a lot.

**Reply by mskocik** [reply]
Take a look at possible solution [REPL](https://svelte.dev/repl/b6016d6613234e8b837e876dc74c4771?version=4.2.11)

**Reply by RvmMim** [reply]
That would work.
But I realized the time picker has not the behavior I was expecting when working with minuteIncrement (it selects the closest minute available instead of the one I'm selecting) so I think I won't use the minuteIncrement for now.
Thanks again and sorry for the inconvenience.

**Reply by mskocik** [reply]
Yes, that's why setting 'disabling' it through css is the only option

**Reply by RvmMim** [reply]
It has this same behavior with the given solution
Update the App.svelte you sent with 
`<script>
	import SveltyPicker from 'svelty-picker@5.2.2';
</script>

<SveltyPicker mode="time" pickerOnly minuteIncrement={15} />

<style>
	:global(
		.is-minute-view button:not([data-value="00"]):not([data-value="30"]):not([data-value="15"]):not([data-value="45"])
	) { 
		opacity: 0.1;
		cursor: default;
	}
</style>`
and you'll see it

**Reply by mskocik** [reply]
You mean that hours can move up and down when you select 00? Or styling, that you can see semitransparent hour values?

**Reply by RvmMim** [reply]
Set minuteIncrement to 15
"Hide" .is-minute-view when data_value = 0 or 15 or 30 or 45
Pick hour
Minute picker is on 0 by default
Click 30 and 15 will be selected, then click 45 and 30 will be selected etc.

And it does the same without the hiding .is-minute-view part

**Reply by mskocik** [reply]
Oh finally we are on the same page! It seems more like a bug, that selected value does not resolve properly.
